qc/homo_sapiens_qc/tennesen_two_pop_qc.py

```diff
@@ -0,0 +1,11 @@
+#%% [markdown]

```

Comment:
I don't think this file does anything; can we remove it?

Reply:
Ah yeah, oops. Delete away.

Reply:
Or actually I'll take care of it